tests/zfs-tests/tests/functional/cache/cache_010_pos.ksh

```diff
@@ -56,29 +56,29 @@ function cleanup_testenv
 	fi
 }
 
-log_assert "Cache device can only be block devices."
+log_assert "Cache device can be files, character devices or block devices."
```

Comment:
The `log_pass` line at the end should match.

Reply:
Done!